Dockerfile

```diff
@@ -1,19 +1,41 @@
-FROM librecores/ci-osstools:2018.1-rc1
+FROM ubuntu:16.04
 
-# make sources available in docker image
-RUN mkdir -p /src
-ADD . /src
-WORKDIR /src
+# travis-ci only provides 2
+ARG MAKE_JOBS=-j2
 
-RUN apt-get update; apt-get install -y virtualenv python3-venv python3-dev
+# Simulation 
+ARG ICARUS_VERILOG_VERSION=10_2 
 
-# Build and test using Python 2
-RUN mkdir /build-py2; cp -r /src /build-py2
-ENV COCOTB=/build-py2/src
-RUN bash -lc 'virtualenv /build-py2/venv; source /build-py2/venv/bin/activate; pip install coverage xunitparser; make -C /build-py2/src test'
+RUN apt-get -qq update && apt-get -qq install -y --no-install-recommends \
+    wget git gperf make autoconf g++ flex bison \
+    python2.7-dev python3-dev\
+    python-pip \
+    python3 virtualenv python3-venv && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get clean && \
+    pip install --upgrade pip && \
+    g++ --version 
```

Comment:
In my experience it's more common to have the `&&` in the next line, as done in the Docker documentation (e.g. here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices). Furthermore, having one line per package makes it easier to extend the list in the future by creating clearer diffs (and reducing merge conflicts)

```
RUN apt-get -qq update \
    && apt-get -qq install -y --no-install-recommends \
        wget \
        git \
        gperf \
        make \
        autoconf \
        g++ \
        flex \
        bison \
        python2.7-dev \
        python3-dev\
        python-pip \
        python3 \
        virtualenv \
        python3-venv \
    && rm -rf /var/lib/apt/lists/* \
    && apt-get clean \
    && pip install --upgrade pip \
    && g++ --version 
```